cl.conf

```diff
@@ -161,15 +161,15 @@ storage:
             sleep 2
           done
 
-
-          export K8S_VERSION=1.9.6
+          export K8S_CTL_VERSION=v1.9.6
+          export K8S_VERSION=v1.9.10
           export MK=/usr/local/bin/minikube
           export KUBECTL=/usr/local/bin/kubectl
           USER_HOME=$(mount | grep nfs | grep -v 'var/amobee\|etc/amobee\|var/log/amobee' | awk '{ print $3 }')
           VAGRANT=${USER_HOME}/.bricks/clusters/vagrant
           
-          sudo curl -Lo $MK https://storage.googleapis.com/minikube/releases/v0.24.1/minikube-linux-amd64 && sudo chmod +x $MK
-          sudo curl -Lo $KUBECTL https://storage.googleapis.com/kubernetes-release/release/v${K8S_VERSION}/bin/linux/amd64/kubectl && sudo chmod +x $KUBECTL
+          sudo curl -Lo $MK https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && sudo chmod +x $MK
+          sudo curl -Lo $KUBECTL https://storage.googleapis.com/kubernetes-release/release/${K8S_CTL_VERSION}/bin/linux/amd64/kubectl && sudo chmod +x $KUBECTL
```

Comment:
I believe the 1.9.10 kubectl version does exist. I just downloaded at that URL.

Reply:
Updated, and tested.